code/modules/mob/living/carbon/give.dm

```diff
@@ -0,0 +1,53 @@
+/mob/living/carbon/verb/give(var/mob/living/carbon/target in oview(1))
+	set category = "IC"
+	set name = "Give"
+
+	if(target.stat == UNCONSCIOUS)
+		to_chat(usr, "<span class='warning'>[target.name] is unconscious!</span>")
+		return
+	else if(target.stat == DEAD)
```

Comment:
I can see this being used to check whether a person is dead or unconscious instead of checking their pulse or whatever. Just check for `target.stat` and tell the user they're not in a state to take an item from them.